packages/angular/executors.json

```
@@ -23,7 +23,7 @@
     "webpack-server": {
       "implementation": "./src/builders/webpack-server/webpack-server.impl",
       "schema": "./src/builders/webpack-server/schema.json",
-      "description": "Serves a browser application with support for a custom webpack configuration."
+      "description": "The webpack-server executor is very similar to the standard dev server builder provided by the Angular Devkit. It is usually used in tandem with @nrwl/angular:webpack-browser when your Angular application uses a custom webpack configuration."
```

nit: I think this would improve the formatting a bit:
```suggestion
      "description": "The webpack-server executor is very similar to the standard dev server builder provided by the Angular Devkit. It is usually used in tandem with `@nrwl/angular:webpack-browser` when your Angular application uses a custom webpack configuration."
```